### PR TITLE
About page frontend

### LINKED ISSUE #111: Implement About Page (frontend)
**Branch:** `about-page-frontend`

### Task at Hand
1. View the [mid-fidelity designs](https://www.figma.com/file/58CoTkFT4TgalsOyc1Patl/%F0%9F%8C%BE-Lagos-Food-Bank?type=design&node-id=1809-21467&mode=design&t=QD3MBT6azmzjOBjZ-0) for the **Admin Edit Website Metadata Flow** pages.
2. Implement the `About` component following the acceptance criteria below.

### Acceptance Criteria
- [ ] Users can see the `About` component with placeholder text in the format of the Figma file.
- [ ] Users can click an edit button which transforms the content of the page into a rich text box that enables typing text, bold/italicized text, and bullet points. The edit button should then be transformed into a save button.
- [ ] After saving their changes, the new page should reflect the changes made in the edit box.

### Notes
- We will need a **rich text editor**. Just using a textbox is not sufficient since it doesn't provide any formatting. The best one I found  is [react-quill](https://github.com/zenoamaro/react-quill). Look into its documentation and see an example [here](https://codesandbox.io/s/react-playground-luctk?file=/src/components/QuillEditor/Editor.jsx).
- Do **not** worry about the cards in the Figma file. Just focus on implementing the text section.
- Use `useState` to save the existing text information. Saving the button should have the page look like the new update (since the updates are saved in a `useState` variable), but refreshing the page will reset the content since we don't make any calls to the backend yet.

### PR DESCRIPTION
**Pull request by snehar27**
## Summary

Closes #111

## Testing

When testing, run the following code:

```
import React from "react";
import About from "@/components/organisms/About";

const Test = () => {
  return <About edit={true} />;
};

export default Test;
```

Then, run `yarn run start` and navigate to `http://localhost:3000/Test`

## Notes

There may be a few importing errors because we had to add to the `package.json` file to implement the rich text editor, but `package.json` was not pushed.

### CODE REVIEW

File: frontend/src/components/organisms/About.tsx [hunk 1]
```diff
@@ -7,12 +11,203 @@ interface AboutProps {
...
+  //           <p>
+  //             <br />
+  //           </p>
+  //         </div>
+
+  //         <script src="https://cdn.quilljs.com/1.3.6/quill.js"></script>
+  //       </div>
+  //     </>
+  //   );
+  // }
```
Comment (akinfelami) [inline frontend/src/components/organisms/About.tsx:210]: please remove all this commented out code?

File: frontend/src/components/organisms/About.tsx [hunk 2]
```diff
@@ -7,12 +11,203 @@ interface AboutProps {
...
+    eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+    ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+    aliquip ex ea commodo consequat. Duis aute irure dolor in
+    reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+    pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+    culpa qui officia deserunt mollit anim id est laborum.
+  </p>
+  <h2>Programs</h2>`;
+  {
+    // <ViewEvents />;
```
Comment (akinfelami) [inline frontend/src/components/organisms/About.tsx:50]: why comment this out? Did you look into what I asked regarding using JSX components in react-quill?

File: frontend/src/components/organisms/ManageUsers.tsx
Comment (akinfelami) [inline frontend/src/components/organisms/ManageUsers.tsx:1]: Please remove this file. It should not be in this PR

File: frontend/src/components/organisms/ViewEvents.tsx
Comment (akinfelami) [inline frontend/src/components/organisms/ViewEvents.tsx:1]: PLEASE remove this file. It should not be in this PR

File: frontend/src/components/organisms/About.tsx [hunk 3]
```diff
@@ -7,12 +11,203 @@ interface AboutProps {
...
+  <p>Fill out this form. Login first</p>
+  `;
+  const [value, setValue] = useState(default_text);
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  <Modal open={open} onClose={handleClose}>
+    <Box>Hello</Box>
+  </Modal>;
```
Comment (akinfelami) [inline frontend/src/components/organisms/About.tsx:73]: Please open the `Test` page where and tell me if you see this part?

File: frontend/src/components/organisms/About.tsx [hunk 4]
```diff
@@ -7,12 +11,203 @@ interface AboutProps {
...
+
+  if (edit == true) {
```
Comment (akinfelami) [inline frontend/src/components/organisms/About.tsx:75]: You don't need this at all. I think I even mentioned to remove the prop entirely from the component. The component is already editable with react-quill

**Review by akinfelami — CHANGES_REQUESTED** [review]
Hey guys!

Please see the changes I requested in this PR

**Review by akinfelami — COMMENTED** [review]
About page is not there. There is still a decision to be made on how we want to format the html + render JSX components together (e.g EventCards). React-quill does not support this out of the box. 
Also it includes file that should not be in this PR. I have attempted to remove to no success

**Review by akinfelami — APPROVED** [review]

**Review by jasozh — APPROVED** [review]

**Pull request merged**